metrics/storage/blogbench_dockerfile/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'blogbench' installed
+
+# Usage: FROM [image name]
+FROM ubuntu
+
+# URL for blogbench test and blogbench version
+ENV BLOGBENCH_URL "https://download.pureftpd.org/pub/blogbench"
+ENV BLOGBENCH_VERSION 1.1
+
+RUN apt-get update && \
+	apt-get install -y build-essential wget && \
```

Comment:
Nit: We appear to be (implicitly) standardising on using `curl` rather than `wget`.